dev-qt/qtnetworkauth/qtnetworkauth-5.9999.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit qt5-build
+
+DESCRIPTION="Network authorization component library for the Qt5 framework"
+LICENSE="GPL-3"
```

Comment:
I refer the ebuild file of qtcharts, that is published under GPL-3 [2]
Could you let me know if I should care more about the license?

[2] http://code.qt.io/cgit/qt/qtcharts.git/tree/